filter/store.go

```diff
@@ -41,11 +43,28 @@ type HTTPClient interface {
 }
 
 // PutCSVData allows the filtered file data to be sent back to the filter store when complete.
-func (store *Store) PutCSVData(filterJobID string, url string, size int64) error {
+func (store *Store) PutCSVData(filterJobID string, fileURL string, size int64) error {
 
-	// todo: make call to the filter API.
+	url := store.filterAPIURL + "/filters/" + filterJobID
+
+	putBody := &observation.Filter{
```

Comment:
My take on it was that its an observation filter, i.e. a filter applied to observations. I am happy to rename though if it does not make sense.